### ISSUE TITLE
svg-to-ts-constants not creating enums

### ISSUE THREAD

**Issue by Bilal-io** [kreuzerk/svg-to-ts#165]
I am seeing an issue where Enums are not created in the file generated from calling `npx -p svg-to-ts svg-to-ts-constants`

My config file:
```
{
  "name": "icons",
  "srcFiles": [
    "./icons/*.svg"
  ],
  "outputDirectory": "./dist/icons/",
  "interfaceName": "Icons",
  "typeName": "icons",
  "prefix": "icons",
  "delimiter": "KEBAB",
  "optimizeForLazyLoading": true,
  "modelFileName": "icons.model",
  "compileSources": false,
  "svgoConfig": {
    "plugins": ["removeDimensions", "cleanupAttrs"]
  },
  "generateType": true,
  "generateEnum": true,
  "enumName": "IconsEnum",
  "generateTypeObject": true,
  "verbose": true
}
```

The interface references the Enum, but the enum itself doesn't exist.
```
export interface Icons {
  name: IconsEnum;
  data: string;
}
```

**Comment by kreuzerk** [comment]
Hi there, we fixed this issue but it's not yet released because we have reached limits on Travis CI free plan and they currently disabled our builds. I think we will need to switch to GitHub actions next and then release this fix.

**Comment by Bilal-io** [comment]
Thank you for your awesome work!
I'd be happy to contribute in any capacity.

**Comment by kreuzerk** [comment]
Hi @Bilal-io thx a lot. Of course, every contribution is welcome. Feel free to take a look at one of the open issues and don't hesitate to reach out to me in case you need some help. Also, I now successfully migrated from Travis to GitHub actions and now successfully released v8.6.0 that contains a fix for this issue.
